include/zephyr/modem/pipe.h

```diff
@@ -130,25 +130,27 @@ void modem_pipe_attach(struct modem_pipe *pipe, modem_pipe_api_callback callback
  * @brief Transmit data through pipe
  *
  * @param pipe Pipe to transmit through
- * @param buf Destination for reveived data
- * @param size Capacity of destination for recevied data
+ * @param buf Data to transmit
+ * @param size Number of bytes to transmit
  *
- * @return Number of bytes placed in pipe
+ * @retval Number of bytes placed in pipe
+ * @retval -EPERM if pipe is closed
+ * @retval -errno code on error
  *
  * @warning This call must be non-blocking
  */
 int modem_pipe_transmit(struct modem_pipe *pipe, const uint8_t *buf, size_t size);
 
 /**
- * @brief Reveive data through pipe
+ * @brief Receive data through pipe
  *
  * @param pipe Pipe to receive from
- * @param buf Destination for reveived data
- * @param size Capacity of destination for recevied data
+ * @param buf Destination for received data; must not be already in use in a modem module.
+ * @param size Capacity of destination for received data
  *
- * @return Number of bytes received from pipe if any
- * @return -EPERM if pipe is closed
- * @return -errno code on error
+ * @retval Number of bytes received from pipe
+ * @retval -EPERM if pipe is closed
```

Comment:
I'd probably follow what's been used in the function desc already and keep the line change minimal, `@return` and `@retval` are use interchangeably in the entire zephyr source with no preference over one another.

Reply:
I made the change from `@return` to `@retval` in order to make the choice uniform in this header.